egs/wsj/s5/steps/nnet3/decode.sh

```diff
@@ -80,7 +80,7 @@ thread_string=
 [ $num_threads -gt 1 ] && thread_string="-parallel --num-threads=$num_threads"
```

Comment:
vijay, I think we can merge this once you make this small change... a few lines above here, add 'utt' at the end of 'sdata'.